### ISSUE TITLE
Why are we loading Microdown?

### ISSUE THREAD

**Issue by ClotildeToullec** [moosetechnology/MooseIDE#995]
It is already present in pharo.
Do we need something outside of Pharo?

**Comment by ClotildeToullec** [comment]
We need the HTMLExporter.
This dependency maybe explains why we have some problems with Microdown.

**Comment by NicolasAnquetil** [comment]
we need it for the "Notebook", no?

But there were plans to transfer this to Pharo

**Comment by LABSARI** [comment]
Yes, we need it for the Notebook. We still have the old Notebook in Moose. We should use the new Notebook.
